Cannot find module './channels' or its corresponding type declarations

### Steps to reproduce
While creating a new app through CLI, it gives error "Cannot find module './channels' or its corresponding type declarations" in both TypeScript and JavaScript.

Following are the steps to reproduce this issue:
Step 1:  Create New App
npm create feathers TestApp

Step 2: Do you want to use JavaScript or TypeScript?
Select Any (JavaScript/TypeScript)

Step 3: Which HTTP framework do you want to use?
I have selected Express, but I think this will not effect the error part.

Step 4: What APIs do you want to offer?
At this step only check HTTP (REST) and not the Real-Time(Socket).

Step 5: Select rest items to default.

------------------------
![image](https://user-images.githubusercontent.com/18046335/224985510-399577b9-414a-498a-a8d5-9b5804d264bc.png)

------------------------

Step 6: Run the application:
npm run dev

It will give the error, because in app.js / app.ts it adds the line for channels but the file is not there. I manually commented the following lines:
- Line Number 21: import { channels } from './channels'
- Line Number 39: app.configure(channels)

### Expected behavior
channel must not be included in app.js/app.ts and run the app properly.

--------------------------------
![image](https://user-images.githubusercontent.com/18046335/224985322-a10af070-814b-477c-bbad-799d48a3ea0f.png)

--------------------------------

--- a/packages/generators/src/app/templates/app.tpl.ts
+++ b/packages/generators/src/app/templates/app.tpl.ts
@@ -81,7 +81,7 @@ ${schema !== false ? `import { configurationValidator } from './configuration'` 
 import { logger } from './logger'
 import { logError } from './hooks/log-error'
 import { services } from './services/index'
-import { channels } from './channels'
+${transports.includes('websockets') ? `import { channels } from './channels'` : ''}
 
 const app: Application = express(feathers())
 

